src/mono/mono/mini/simd-intrinsics-netcore.c

```diff
@@ -1158,48 +1157,62 @@ emit_x86_intrinsics (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature
 		case SN_Insert:
 			g_assert (arg0_type == MONO_TYPE_I2 || arg0_type == MONO_TYPE_U2);
 			return emit_simd_ins_for_sig (cfg, klass, OP_XINSERT_I2, 0, arg0_type, fsig, args);
-		case SN_ShiftRightLogical:
-			if (fsig->params [1]->type == MONO_TYPE_U1) {
-				if (arg0_type == MONO_TYPE_I2 || arg0_type == MONO_TYPE_U2)
-					return emit_simd_ins_for_sig (cfg, klass, OP_SSE2_PSRLW_IMM, -1, arg0_type, fsig, args);
-				if (arg0_type == MONO_TYPE_I4 || arg0_type == MONO_TYPE_U4)
-					return emit_simd_ins_for_sig (cfg, klass, OP_SSE2_PSRLD_IMM, -1, arg0_type, fsig, args);
-				if (arg0_type == MONO_TYPE_I8 || arg0_type == MONO_TYPE_U8)
-					return emit_simd_ins_for_sig (cfg, klass, OP_SSE2_PSRLQ_IMM, -1, arg0_type, fsig, args);
-				else {
-					g_assert_not_reached ();
-					break;
-				}
-			} else {
-				// FIXME:
-				return NULL;
+		case SN_ShiftRightLogical: {
```

Comment:
SN_ShiftRightLogical
SN_ShiftRightArithmetic
SN_ShiftLeftLogical
were implemented only for "imm" overloads.